src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -19,11 +19,12 @@ public Movie add(Movie movie) {
 
     @Override
     public Movie get(Long id) {
-        return movieDao.get(id).get();
+        return movieDao.get(id).orElseThrow(() -> new RuntimeException("No value"
+                + " found for Movie id:" + id));
```

Comment:
Consider using a more specific exception instead of a generic RuntimeException to provide more context about the error. 🚀